getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>It would be helpful to mention what the matrices actually do.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>